route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -5,6 +5,7 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Security.Cryptography;
```

Comment:
why do we need Cryptography here?

Reply:
We do not need, it remained from old case. We can remove it.